plugins/permission-node/src/integration/createPermissionIntegrationRouter.ts

```diff
@@ -194,6 +202,7 @@ export const createPermissionIntegrationRouter = <
       name: rule.name,
       description: rule.description,
       resourceType: rule.resourceType,
+      schema: zodToJsonSchema(rule.schema),
       parameters: {
```

Comment:
Perhaps we should drop this property entirely, since we're already making a breaking change? It's always going to be 1 from now on anyway.

Reply:
Removed!